a-seed-from-nothing.sh

```diff
@@ -84,6 +77,7 @@ fi
 set -e
 
 # Ensure an ssh key is generated
+CLOUD_USER=$(ls /home | grep -v lab | grep -v stack | head -1)
```

Comment:
A bit unsure about this one.

Reply:
Yeah, it's a bit wacky but it does work. I just needed to get something working on the Friday before I left for holiday. I'll take another look when I get a chance